DB/src/main/java/io/deephaven/db/v2/select/AbstractConditionFilter.java

```diff
@@ -152,13 +146,42 @@ public synchronized void init(TableDefinition tableDefinition) {
             }
             params = paramsList.toArray(Param.ZERO_LENGTH_PARAM_ARRAY);
 
+            // check if this is a filter that uses a numba vectorized function
+            for (Param param : params) {
+                if (param.getValue().getClass() == NumbaCallableWrapper.class) {
+                    NumbaCallableWrapper numbaCallableWrapper = (NumbaCallableWrapper) param.getValue();
+                    DeephavenCompatibleFunction dcf = DeephavenCompatibleFunction.create(numbaCallableWrapper.getPyObject(),
+                            numbaCallableWrapper.getReturnType(), usedColumns.toArray(new String[0]),
+                            true);
+                    checkReturnType(result, dcf.getReturnedType());
+                    setFilter(new ConditionFilter.ChunkFilter(
+                            dcf.toFilterKernel(),
+                            dcf.getColumnNames().toArray(new String[0]),
+                            ConditionFilter.CHUNK_SIZE));
+                    initialized = true;
+                    return;
+                }
+            }
```

Comment:
I'm confused by this - looks like it's possible to return early, before we've investigated all of the parameters. What happens if there are multiple params that are NumbaCallableWrapper?